base_layer/core/src/transactions/transaction.rs

```diff
@@ -638,6 +649,31 @@ impl Transaction {
     pub fn min_spendable_height(&self) -> u64 {
         max(self.max_kernel_timelock(), self.max_input_maturity())
     }
+
+    // Note this will also do cut-through
+    pub fn add_no_cut_through(mut self, other: Self) -> Self {
```

Comment:
The comment contradicts the function name

Reply:
copy paste issue, fixed